python/cudf/cudf/tests/test_array_ufunc.py

```diff
@@ -70,6 +91,13 @@ def test_ufunc_index(request, ufunc):
             reason=f"cupy has no support for '{fname}'",
         )
     )
+    request.applymarker(
+        pytest.mark.xfail(
+            condition=fname == "matmul"
+            and version.parse(pd.__version__) < version.parse("3.0"),
```

Comment:
I guess this is an argument for having `PANDAS_LT_300`?

Reply:
Yup, added a `PANDAS_LT_300` here.